server/routes/items.js

```diff
@@ -132,8 +145,55 @@ router.post(
         processorRequest
       );
       processorToken = processorTokenResponse.data.processor_token;
+
+      // create Dwolla Customer and obtain customer url
+      await axios
+        .post(
+          'https://api-sandbox.dwolla.com/customers',
```

Comment:
might want to put this as a constant instead like `const DWOLLA_BASE_URL = 'https://api-sandbox.dwolla.com'` and then you can just do something like `${DWOLLA_BASE_URL}/customers` (and do the same for the other dwolla endpoints)

Reply:
ooh...good idea! thanks!